src/systemd/sd-id128.h

```diff
@@ -119,6 +120,37 @@ _sd_pure_ static __inline__ int sd_id128_is_allf(sd_id128_t a) {
 #define SD_ID128_NULL ((const sd_id128_t) { .qwords = { 0, 0 }})
 #define SD_ID128_ALLF ((const sd_id128_t) { .qwords = { UINT64_C(0xFFFFFFFFFFFFFFFF), UINT64_C(0xFFFFFFFFFFFFFFFF) }})
 
+_sd_pure_ static __inline__ int sd_id128_equalv(sd_id128_t a, va_list ap) {
+        int r = 0;
+
+        for (;;) {
+                sd_id128_t b = va_arg(ap, sd_id128_t);
+
+                if (sd_id128_is_null(b))
+                        break;
+
+                r = sd_id128_equal(a, b);
+                if (r)
+                        break;
```

Comment:
```c
if (sd_id128_equal(…))
        return true;
```

i mean, you can get rid of `r` here, if you just exit the loop immediately

Reply:
Indeed.